internal/driver/commands.go

```diff
@@ -129,115 +129,115 @@ var pprofCommands = commands{
 // reported generated by pprof.
 var pprofVariables = variables{
 	// Filename for file-based output formats, stdout by default.
-	"output": &variable{stringKind, "", "", helpText("Output filename for file-based outputs")},
+	"output": &variable{stringKind, []string{""}, "", helpText("Output filename for file-based outputs")},
```

Comment:
I don't think it would be much harder do tag filtering on regular expression, if that's why you don't want to do regex matches.
I liked the double equals sign concept better when a user would list tagfocus multiple times
somthing about '-tagfocus=key1=value1,key2=value2' looks weird to me. 
I think using slashes should be good. I'm assuming both "tagfocus=/mytag/128kb:512kb" and "tagfocus=mytag/128kb:512kb" should be valid?

Do we only want the slashes to work for numeric labels, or for both types of labels?

I'm assuming something we'd keep the and-like notation, but accept a sample as a match to 
tagfocus=/mytag/128kb:512kb even if it had no tag mytag?